### PR TITLE
Updated SET 14 of resources.md

### LINKED ISSUE #33: SET 14 Django Tutorials points to the wrong link!
**Describe the bug/feature**

The Django Tutorials label points rather to a Flask Tutorial than Django.
Link: https://codezonediitj.github.io/resources/
**To Reproduce(for bug)**

**Screenshots(for bug)**

SiteLabel
TutorialPointedto

**Device Details(for bug)**

**Other Comments**

Suggested change would be to change the Label to Flask Tutorial rather.

### PR DESCRIPTION
**Pull request by harshaldesai01**
#### Type of Change:

- [x] Code
- [ ] Blog
- [x] Resources

#### Description
Changed the label from "Django Material" to "Flask Tutorial"
Fixes #33

### CODE REVIEW

File: resources.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ This page contains resources for computer science. If you too have something to
 </div>
 <div class="box" markdown="1" id="14">
 ### Set 14
-[Django Material](https://www.youtube.com/watch?v=MwZwr5Tvyxo&list=PL-osiE80TeTs4UjLw5MM6OjgkjFeUxCYH)<br/>
+[Flask Tutorial](https://www.youtube.com/watch?v=MwZwr5Tvyxo&list=PL-osiE80TeTs4UjLw5MM6OjgkjFeUxCYH)<br/>
 [Abhishek Nanda](https://www.facebook.com/NandaAbhishek118/)<br/>
```
Comment (czgdp1807) [inline resources.md]: I think L121 can be shifted to the contributors section at the bottom of the page. Can you please make that change?

**Comment by czgdp1807** [comment]
Can you please change, `Fixes # [ISSUE NUMBER HERE]` to `Fixes #ISSUE_NUMBER_HERE` in [this file](https://github.com/codezonediitj/codezonediitj.github.io/blob/81d09fb494ac731f472de4b794205c5900f8e9c1/PULL_REQUEST_TEMPLATE.md)? A separate PR will have to be made for this trivial change. It will be great if you do the change directly via this PR. Thanks.

File: resources.md (hunk 1 above)
Comment (harshaldesai01) [reply resources.md]: Sure!

**Comment by harshaldesai01** [comment]
> Can you please change, `Fixes # [ISSUE NUMBER HERE]` to `Fixes #ISSUE_NUMBER_HERE` in [this file](https://github.com/codezonediitj/codezonediitj.github.io/blob/81d09fb494ac731f472de4b794205c5900f8e9c1/PULL_REQUEST_TEMPLATE.md)? A separate PR will have to be made for this trivial change. It will be great if you do the change directly via this PR. Thanks.

Okay!

**Comment by harshaldesai01** [comment]
Done with the changes, you can review and merge now!

**Pull request merged**